workflow/rocoto/gfs_tasks.py

```diff
@@ -553,17 +553,31 @@ def _fcst_cycled(self):
 
     def post(self):
         add_anl_to_post = False
-        if self.app_config.mode in ['cycled']:
-            add_anl_to_post = True
-
         return self._post_task('post', add_anl_to_post=add_anl_to_post)
 
+    def postanl(self):
```

Comment:
On lines 555 and 556, please remove the argument `add_anl_to_post`.  It is no longer needed.